### DISCUSSION TITLE
v3 How to use pretrained weights in training NER model ?

### DISCUSSION THREAD

**Discussion by ksingh196** [explosion/spaCy#7021] [Help: Coding & Implementations] [answered]
I have pretrained weights for tok2vec but how to use these while training the NER model when using commandline
I have tried  --init-tok2vec, -t2v but i just get Error: no such option:.

Has anybody faced this in V3

**Comment by svlandeg** [comment, marked as answer]
This mechanism was pretty experimental in v2, but has matured quite a bit in v3. One of the problems we saw was that it was difficult to ensure that all parameters between pretraining and training were the same. To streamline this, the pretraining step can now be specificied in the configuration file, and then you'd basically use the same configuration file for training as well as pretraining. You can find more details in the docs here: https://spacy.io/usage/embeddings-transformers#pretraining and here: https://spacy.io/api/cli#pretrain

**Comment by ksingh196** [comment]
Thanks it worked.
